### DISCUSSION TITLE
Failed to load script: /_next/static/chunks/pages...

### DISCUSSION THREAD

**Discussion by mattjs** [vercel/next.js#32119] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

14.17.6

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

node command

### Describe the Bug

In development, I have started running into more and more of these `Failed to load script` errors when clicking links

<img width="1001" alt="Screen Shot 2021-12-03 at 4 31 20 PM" src="https://user-images.githubusercontent.com/2204775/144681816-e8a5e75c-9dab-42f3-a9da-1e04f771a3e3.png">

After the error shows up, the page then loads correctly.

### Expected Behavior

The page to load without error.

### To Reproduce

Described above. For some links it happens and for others it doesn't. More specifically dynamic links.

**Comment by timneutkens** [comment]
You'll want to upgrade to the latest version. Converted to discussions given that no full reproduction was provided.

**Reply by mattjs** [reply]
Hey Tim. Upgrading to 12 gave issues with existing CommonJS modules we are using along with rejecting some third party `styled-jsx` (I'm guessing because it was partially invalid). Though all of the stuff in 12 seems great its hard to make time for this migration at the moment.

**Reply by NicolasPineau** [reply]
Just faced same issue on 12.0.4, it was solved after upgrading to 12.0.7.

**Reply by Lasni** [reply]
This is still an issue in 13.0.1

**Reply by vikito755** [reply]
Issue persisting in Next 13.4.16.

**Reply by SimplySayHi** [reply]
Same issue in 13.5.4
